crates/loader/tests/ui/invalid-digest.err

```diff
@@ -0,0 +1,6 @@
+Failed to load Spin app from "<test-dir>/invalid-digest.toml"
+
+Caused by:
+    0: Failed to load component `web`
+    1: Failed to load Wasm source Remote { url: "https://example.com/wasm.wasm.wasm", digest: "not-a-digest" }
```

Comment:
nit: can we improve the error message to not print the debug output of the `Remote` type? Maybe like:

Failed to load Wasm source: https://example.com/wasm.wasm.wasm with digest: "not-a-digest"